airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -672,6 +672,7 @@ def invoke_defer_method(self):
         )
 
     def execute_complete(self, context: Context, event: dict, **kwargs):
+        self.log.info("Triggered with event: %s", event)
```

Comment:
Shouldn't this be debug?

Reply:
Thanks for raising the question! But I wanted to enable it by default because 1) it is a low frequency message, it is printed at most once per task; 2) when error happens, it is very useful for debugging. I wanted to avoid additional steps to enable debug logs and rerun.

Let me know if you still think debug is the right logging level.

Reply:
Yeah. I thin in this case debug  might be a better level, depending on the details printed in the `event` dict - if they are big and long (as I presume) - then such a log entry should definitely be debug.

Suggestion maybe Composer team could introduce some ways of changing the log level by the customer remotely or dynamically when needed ?  Actually I think it could be a good idea to add a flag in the UI to do "rerun with debug" - that could be a very nice feature to add to Airflow actually, if Composer tem would find it useful, maybe a good idea to contribute such feature?

Reply:
Updated log level to debug.